tests/migration/reset_scc_addons_values.pm

```diff
@@ -0,0 +1,64 @@
+# Copyright 2023 SUSE LLC
+# SPDX-License-Identifier: FSFAP
+#
+# Summary: Swap the values of SCC_ADDONS before migration.
+# In MU jobs, the SCC_ADDONS is used to add the products' MU repos;
+# In migration jobs, the setting is also used to register modules.
+# On SLES12SPx and SLES15SPx, the modules name are different, so need
+# to swap SCC_ADDONS values to sles15SP5 modules nemes before migration.
+#
```

Comment:
we need to simplify this file (at the current state the code is not really maintainable).
First idea (already mentioned) to set SCC_ADDONS for the target product with a settings,  `SCC_ADDONS_2` or `SCC_ADDONS_TEMP_2`. We don't need to calculate the value.

Reply:
Updated, thanks for review.